src/main/webapp/client/views/index.html

```diff
@@ -28,26 +28,27 @@
 		class="navbar navbar-expand-lg navbar-dark ftco_navbar bg-dark ftco-navbar-light"
 		id="ftco-navbar">
 		<div class="container">
-			<a class="navbar-brand" href="index.html">Jobpply</a>
+			<a class="navbar-brand" href="index.html" th:text="#{client.index.homepageLabel}">Jobpply</a>
 			<button class="navbar-toggler" type="button" data-toggle="collapse"
 				data-target="#ftco-nav" aria-controls="ftco-nav"
-				aria-expanded="false" aria-label="Toggle navigation">
+				aria-expanded="false" aria-label="Toggle navigation" 
+				th:text="#{client.index.btnMenu}"> 
 				<span class="oi oi-menu"></span> Menu
 			</button>
 
 			<div class="collapse navbar-collapse" id="ftco-nav">
 				<ul class="navbar-nav ml-auto">
 					<li class="nav-item active"><a href="index.html"
-						class="nav-link">Home</a></li>
-					<li class="nav-item"><a href="about.html" class="nav-link">About</a></li>
+						class="nav-link" th:text="#{client.index.navLinkHome}">Home</a></li> 
+					<li class="nav-item"><a href="about.html" class="nav-link" th:text="#{client.index.navLinkAbout}">About</a></li> 
 					<li class="nav-item"><a href="candidates.html"
-						class="nav-link">Canditates</a></li>
-					<li class="nav-item"><a href="blog.html" class="nav-link">Blog</a></li>
-					<li class="nav-item"><a href="contact.html" class="nav-link">Contact</a></li>
+						class="nav-link" th:text="#{client.index.navLinkCandidate}">Canditates</a></li> 
+					<li class="nav-item"><a href="blog.html" class="nav-link" th:text="#{client.index.navLinkBlog}">Blog</a></li> 
+					<li class="nav-item"><a href="contact.html" class="nav-link" th:text="#{client.index.navLinkContact}">Contact</a></li> 
 					<li class="nav-item cta mr-md-1"><a href="new-post.html"
-						class="nav-link">Post a Job</a></li>
+						class="nav-link" th:text="#{client.index.navLinkPostJob}">Post a Job</a></li> 
 					<li class="nav-item cta cta-colored"><a href="job-post.html"
-						class="nav-link">Want a Job</a></li>
+						class="nav-link" th:text="#{client.index.navLinkWantJob}">Want a Job</a></li> 
```

Comment:
`Want a Job` code này có thừa ko chú

Reply:
ý chị là thừa chỗ cái text "Want a Job" đúng ko ạ?
Do em đọc document của thymeleaf em toàn thấy họ code theo kiểu vừa có th:text, vừa vẫn để text đó nên em bắt chước vậy ạ. Mặc dù em ko biết để thế làm gì :D 
https://www.thymeleaf.org/doc/tutorials/3.0/usingthymeleaf.html#using-texts

Reply:
để nếu ko có load dc biến th:text thì nó dùng tạm cũng nên, thử phát nào

Reply:
không load được biến thì nó trả về ??key?? chị ạ, còn muốn set default thì em tìm được cách code siêu dài siêu loằng ngoằng như này chị :v 
https://stackoverflow.com/questions/35115860/thymeleaf-internationalization-use-default-text-html/35160518